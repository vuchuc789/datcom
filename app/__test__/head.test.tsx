import { render } from '@testing-library/react';
import Head from '../head';

describe('Root Head', () => {
  test('render correctly', () => {
    const { container } = render(<Head />);

    expect(container).toMatchSnapshot();
  });
});

export {};
