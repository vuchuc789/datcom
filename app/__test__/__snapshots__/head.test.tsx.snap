// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Head render correctly 1`] = `
<div>
  <title>
    Cơm Cơm
  </title>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    content="Generated by create next app"
    name="description"
  />
  <link
    href="/favicon.ico"
    rel="icon"
  />
</div>
`;
